src/applications/gi/components/search/KeywordSearch.jsx

```diff
@@ -54,7 +54,11 @@ export class KeywordSearch extends React.Component {
       event: 'gibct-autosuggest',
       'gibct-autosuggest-value': searchQuery,
     });
-    this.props.onFilterChange(searchQuery);
+    if (searchQuery) {
```

Comment:
Why are checking on the value of `searchQuery`?

Why not have this just be behavior that is enabled by setting a property to true?